test/distributed/org/apache/cassandra/fuzz/sai/AccordMultiNodeSAITest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.fuzz.sai;
+
+import org.junit.BeforeClass;
+
+public class AccordMultiNodeSAITest extends MultiNodeSAITestBase
```

Comment:
The test itself looks fine, but I'm getting some timeouts locally closing the cluster. Notable in the thread dump are:

```
Threads: [node1_isolatedExecutor:11]:
	java.base@11.0.19/jdk.internal.misc.Unsafe.park(Native Method)
	java.base@11.0.19/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:234)
	org.apache.cassandra.simulator.systems.InterceptorOfSystemMethods$None.parkNanos(InterceptorOfSystemMethods.java:373)
	org.apache.cassandra.simulator.systems.InterceptorOfSystemMethods$Global.parkNanos(InterceptorOfSystemMethods.java:166)
	java.base@11.0.19/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2123)
	java.base@11.0.19/java.util.concurrent.ThreadPoolExecutor.awaitTermination(ThreadPoolExecutor.java:1454)
	org.apache.cassandra.service.accord.api.AccordScheduler.awaitTermination(AccordScheduler.java:101)
	org.apache.cassandra.utils.ExecutorUtils.shutdownSequentiallyAndWait(ExecutorUtils.java:98)
	org.apache.cassandra.service.accord.AccordService.shutdown(AccordService.java:990)
	org.apache.cassandra.service.accord.AccordService.shutdownAndWait(AccordService.java:1024)
```

```
Threads: [node2_AccordJournal-allocator]:
	java.base@11.0.19/jdk.internal.misc.Unsafe.park(Native Method)
	java.base@11.0.19/java.util.concurrent.locks.LockSupport.park(LockSupport.java:323)
	org.apache.cassandra.utils.concurrent.WaitQueue$Standard$AbstractSignal.await(WaitQueue.java:290)
	org.apache.cassandra.utils.concurrent.WaitQueue$Standard$AbstractSignal.await(WaitQueue.java:283)
	org.apache.cassandra.utils.concurrent.WaitQueue.waitOnCondition(WaitQueue.java:434)
	org.apache.cassandra.journal.Journal$AllocateRunnable.runNormal(Journal.java:674)
	org.apache.cassandra.journal.Journal$AllocateRunnable.run(Journal.java:627)
	org.apache.cassandra.concurrent.InfiniteLoopExecutor.loop(InfiniteLoopExecutor.java:126)
```

```
Threads: [node1_AccordJournal-disk-flusher-periodic]:
	java.base@11.0.19/jdk.internal.misc.Unsafe.park(Native Method)
	java.base@11.0.19/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:234)
	org.apache.cassandra.simulator.systems.InterceptorOfSystemMethods$None.parkNanos(InterceptorOfSystemMethods.java:373)
	org.apache.cassandra.simulator.systems.InterceptorOfSystemMethods$Global.parkNanos(InterceptorOfSystemMethods.java:166)
	java.base@11.0.19/java.util.concurrent.locks.AbstractQueuedSynchronizer.doAcquireSharedNanos(AbstractQueuedSynchronizer.java:1079)
	java.base@11.0.19/java.util.concurrent.locks.AbstractQueuedSynchronizer.tryAcquireSharedNanos(AbstractQueuedSynchronizer.java:1369)
	java.base@11.0.19/java.util.concurrent.Semaphore.tryAcquire(Semaphore.java:592)
	org.apache.cassandra.utils.concurrent.Semaphore$Standard.tryAcquireUntil(Semaphore.java:144)
	org.apache.cassandra.journal.Flusher$FlushRunnable.doRun(Flusher.java:294)
	org.apache.cassandra.journal.Flusher$FlushRunnable.run(Flusher.java:254)
	org.apache.cassandra.concurrent.InfiniteLoopExecutor.loop(InfiniteLoopExecutor.java:126)
```

Reply:
Same thing if you run `FullAccordCQLTest`